src/main/groovy/com/bmuschko/gradle/docker/DockerRegistryCredentials.groovy

```diff
@@ -116,4 +116,14 @@ class DockerRegistryCredentials {
             }
         }
     }
+
+    /**
+     * Check if credentials are valid. Valid means that it has
+     * at least one of the following properties set:
+     * username, password, email
+     * @return
```

Comment:
This portion of the Javadocs doesn't actually describe the return. Needs to have a description.